github/orgs_actions_allowed.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// ActionsAllowed represents selected actions that are allowed in an organization
+// GitHub API docs: https://docs.github.com/en/rest/reference/actions#get-allowed-actions-for-an-organization
+type ActionsAllowed struct {
+	GithubOwnedAllowed *bool     `json:"github_owned_allowed,omitempty"`
+	VerifiedAllowed    *bool     `json:"verified_allowed,omitempty"`
+	PatternsAllowed    *[]string `json:"patterns_allowed,omitempty"`
+}
+
+func (a ActionsAllowed) String() string {
+	return Stringify(a)
+}
+
+// GetActionsAllowed gets a specific artifact for a workflow run.
+//
+// GitHub API docs: https://docs.github.com/en/rest/reference/actions#get-allowed-actions-for-an-organization
+func (s *OrganizationsService) GetActionsAllowed(ctx context.Context, org string) (*ActionsAllowed, *Response, error) {
+	u := fmt.Sprintf("orgs/%v/actions/permissions/selected-actions", org)
+
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	actionsAllowed := new(ActionsAllowed)
+	resp, err := s.client.Do(ctx, req, actionsAllowed)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return actionsAllowed, resp, nil
+}
+
+// EditActionsAllowed sets the actions that are allowed in an organization
+//
+// GitHub API docs: https://docs.github.com/en/rest/reference/actions#set-allowed-actions-for-an-organization
+func (s *OrganizationsService) EditActionsAllowed(ctx context.Context, org string, actionsAllowed *ActionsAllowed) (*ActionsAllowed, *Response, error) {
```

Comment:
```suggestion
func (s *OrganizationsService) EditActionsAllowed(ctx context.Context, org string, actionsAllowed ActionsAllowed) (*ActionsAllowed, *Response, error) {
```

Reply:
Thanks @gmlewis for the prompt review, much appreciated!
I'm happy to apply this change, just wanted to double-check first. This pattern of passing pointers to the Edit methods has been used in other files of this library (e.g. https://github.com/google/go-github/blob/master/github/orgs.go#L230), is this something we'd like to change anyway?

Reply:
I'm trying to follow the pattern of passing-by-value for required parameters so that a `nil` does not get passed.

In my opinion, we should try to follow this pattern throughout this repo's API, but I have unfortunately not always been consistent in my code reviews.

Therefore, I recommend we go ahead and pass required parameters by value here in this case, but not break the current API in the other cases... and maybe long-term we should have a single cleanup PR that makes the pattern consistent throughout the repo (after we give users of this repo a chance to discuss the pros/cons of breaking the existing API).

Reply:
Makes sense, I'll update the PR with the changes 👍